code/datums/spells/mimic.dm

```diff
@@ -19,7 +19,7 @@
 	var/perfect_disguise = FALSE
 
 	var/static/list/black_listed_form_types = list(/obj/screen, /obj/singularity, /obj/effect, /mob/living/simple_animal/hostile/megafauna, /atom/movable/lighting_object, /obj/machinery/dna_vault,
-													/obj/machinery/power/bluespace_tap, /obj/structure/sign/barsign, /obj/machinery/atmospherics/unary/cryo_cell)
+													/obj/machinery/power/bluespace_tap, /obj/structure/sign/barsign, /obj/machinery/atmospherics/unary/cryo_cell, /obj/machinery/gravity_generator)
```

Comment:
These are long lines that don't quite fit on the screen.
With so many blacklisted things, i think it's worth splitting the list to one item per line. Something like

```dm
	var/static/list/black_listed_form_types = list(
		/obj/screen,
		/obj/singularity,
		/obj/effect,
		/mob/living/simple_animal/hostile/megafauna,
		/atom/movable/lighting_object,
		/obj/machinery/dna_vault,
		/obj/machinery/power/bluespace_tap,
		/obj/structure/sign/barsign,
		/obj/machinery/atmospherics/unary/cryo_cell,
		/obj/machinery/gravity_generator
	)
```

Reply:
Done!